integrations-and-supported-tools/detectron2/notebooks/Neptune_detectron2.ipynb

```diff
@@ -0,0 +1,490 @@
+    "    api_token=getpass(\"Enter your Neptune API token: \"),\n",
+    "    project=\"workspace-name/project-name\",  # replace with your own\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "For example, if your workspace name is `ml-team` and the project name is `classification`, the project argument is: `project=\"ml-team/classification\"`.\n",
+    "\n",
+    "To find your API token and project name, [log in to Neptune](https://app.neptune.ai/).\n",
+    "- In the top-right corner, click your avatar and select **Get your API token**.\n",
+    "- To find and copy your project name, navigate to the project, then click **Settings** → **Properties**."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "q3Hm8xjsr5Hn",
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "---\n",
+    "\n",
+    "You now have new run in Neptune! From here on, we'll use the `run` object to log metadata.\n",
+    "\n",
+    "**To open the run in Neptune, follow the link that appeared in the cell output.**\n",
+    "\n",
+    "There's not much to display yet, but keep the tab with the run open to see what happens next."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "PoU3wE8UyivB"
+   },
+   "source": [
+    "## Log metadata to Neptune"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch, detectron2\n",
+    "\n",
+    "import detectron2\n",
+    "from detectron2.utils.logger import setup_logger\n",
+    "\n",
+    "setup_logger()\n",
+    "\n",
+    "# import some common libraries\n",
+    "import numpy as np\n",
+    "import os, json, cv2, random\n",
+    "\n",
+    "# import some common detectron2 utilities\n",
+    "from detectron2 import model_zoo\n",
+    "from detectron2.engine import DefaultPredictor\n",
+    "from detectron2.config import get_cfg\n",
+    "from detectron2.utils.visualizer import Visualizer\n",
+    "from detectron2.data import MetadataCatalog, DatasetCatalog"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "PT-fWITQxJMw"
+   },
+   "outputs": [],
+   "source": [
+    "# step code\n",
+    "# download, decompress the data\n",
+    "! wget https://github.com/matterport/Mask_RCNN/releases/download/v2.1/balloon_dataset.zip\n",
+    "! unzip balloon_dataset.zip > /dev/null"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5qagCeeazjrW"
+   },
+   "source": [
+    "### Transform dataset for training"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "C36-L8cUzjrW"
+   },
+   "outputs": [],
+   "source": [
+    "# step code\n",
+    "# if your dataset is in COCO format, this cell can be replaced by the following three lines:\n",
```

Comment:
Maybe this part of the comment (lines 237-240) would be nicer to have in a proper text cell?

Reply:
Removed this as it was irrelevant for our example.